config/config.go

```diff
@@ -256,6 +253,60 @@ func (c *Auth) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+type MetricOverrides struct {
+	StringMetrics map[string][]StringMetric `yaml:"string_metrics,omitempty"`
```

Comment:
I don't think string_metrics quite captures it, we have other string metrics. regexp_extract or similar would be better.